src/ManageInformationView.elm

```diff
@@ -0,0 +1,152 @@
+                            [ button
+                                [ onClick (DeleteSavedFile f.id)
+                                , class "button button--secondary icon icon--bin"
+                                ]
+                                []
+                            ]
+                        , div [ class "constructor__question-controls" ]
+                            [ button
+                                [ class "button button--glass icon icon--up-open"
+                                , onClick (MoveFileUp i)
+                                , disabled (i == 0)
+                                ]
+                                []
+                            , button
+                                [ class "button button--glass icon icon--down-open"
+                                , onClick (MoveFileDown i)
+                                , disabled (i == ((List.length model.savedFiles) - 1))
+                                ]
+                                []
+                            ]
+                        ]
+                )
+                model.savedFiles
+
+        filesToSaveDivs =
+            model.filesToSave
+                |> List.map
+                    (\f ->
+                        div []
+                            [ div [ class "form__question-sub-section form__question-sub-section--table" ]
+                                [ input
+                                    [ class "form__input"
+                                    , type_ "text"
+                                    , placeholder "File title"
+                                    , id ("file-title-" ++ (toString f.id))
+                                    , onInput (ChangeFileToSaveTitle f.id)
+                                    ]
+                                    []
+                                ]
+                            , div [ class "form__question-sub-section form__question-sub-section--table" ]
+                                [ input
+                                    [ id (toString f.id)
+                                    , type_ "file"
+                                    , on "change"
+                                        (Json.Decode.succeed (FileSelected f.id))
+                                    ]
+                                    []
+                                ]
+                            , div [ class "form__question-sub-section form__question-sub-section--table form__question-sub-section--button" ]
+                                [ button
+                                    [ onClick (DeleteFileToSave f.id)
+                                    , class "button button--secondary icon icon--bin"
+                                    ]
+                                    []
+                                ]
+                            ]
+                    )
+
+        displayWarning =
+            if not (String.isEmpty (manageInformationWarning model)) then
```

Comment:
can simply be `not (String.isEmpty (manageInformationWarning model))`

Reply:
👍